vision/classification/efficientnet-lite4/README.md

```diff
@@ -0,0 +1,108 @@
+# EfficientNet-Lite4
+
+## Use-Cases
+EfficientNet-Lite4 is an image classification model that achieves state-of-the-art accuracy. It is designed to run on mobile CPU, GPU, and EdgeTPU devices, allowing for applications on mobile and loT, where computational resources are limited.
+
+## Description
+EfficientNet-Lite4 is a version of EfficentNet-Lite, where it is the largest variant, integer-only quantized model that produces the highest accuracy. ImageNet top-1 accuracy, while still running in real-time (e.g. 30ms/image) on a Pixel 4 CPU. 
+
+## Model
+
+ |Model        |Download | Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|EfficientNet-Lite4     | [51.9 MB](model/efficientnet-lite4.onnx)	  | [157 MB](efficientnet-lite4.tar.gz)|1.7.0|11|
+
+
+
+<hr>
+
+## Inference
+
+### Input to model
+Input to model has the image shape float32[1,224,224,3]. Inference was done using a jpg image.
```

Comment:
describe the dimensions (batch size, height, width, channels)? You should double check and confirm.